common/static/common/js/spec/components/list_spec.js

```diff
@@ -0,0 +1,67 @@
+define(['jquery', 'backbone', 'underscore', 'common/js/components/views/list'], function ($, Backbone, _, ListView) {
```

Comment:
Should `'use strict';`. Also, redoing the formatting would be nice (second argument on a new line).